src/resources/elements/primeDesignSystem/ptextarea/ptextarea.ts

```diff
@@ -7,11 +7,12 @@ import {ValidationState} from "../types";
 @customElement("ptextarea")
 export class PTextarea {
 
-  @bindable public validationState?: ValidationState;
+  @bindable validationState?: ValidationState;
 
-  @bindable public autocomplete = "off";
-  @bindable.booleanAttr public disabled;
-  @bindable.number public rows = 4;
-  @bindable({defaultBindingMode: bindingMode.twoWay}) public value: string | BigNumber;
-  @bindable.string public placeholder = "";
+  @bindable autocomplete = "off";
+  @bindable.booleanAttr disabled;
+  @bindable.number rows = 4;
+  @bindable({defaultBindingMode: bindingMode.twoWay}) value: string | BigNumber;
+  @bindable.string placeholder = "";
+  @bindable.booleanAttr resizable = true;
```

Comment:
Test this carefully.  In my experience `@bindable.booleanAttr` only works the way we expect if its default value is `false`

Reply:
me and @paweljanicki tested this on the [storybook page](https://prime-deals-dapp-git-pform-input-display-block-curvelabs.vercel.app/storybook/ptextarea) and it seems it works as expected. Do you have a specific edge-case we should check for?

Reply:
ok I don't recall, lets leave it